packages/gcf-utils/package.json

```diff
@@ -23,14 +23,14 @@
     "@google-cloud/secret-manager": "^5.6.0",
     "@google-cloud/storage": "^7.12.1",
     "@google-cloud/tasks": "^5.5.0",
-    "@octokit/auth-app": "^4.0.5",
-    "@octokit/core": "^5.0.2",
-    "@octokit/graphql": "^5.0.0",
-    "@octokit/plugin-enterprise-compatibility": "^2.0.3",
-    "@octokit/plugin-paginate-rest": "9.2.2",
+    "@octokit/auth-app": "^7.1.5",
+    "@octokit/core": "^5.2.0",
+    "@octokit/graphql": "^8.2.1",
+    "@octokit/plugin-enterprise-compatibility": "^5.0.1",
+    "@octokit/plugin-paginate-rest": "^9.2.2",
     "@octokit/request": "^8.4.1",
     "@octokit/request-error": "^5.1.1",
-    "@octokit/rest": "^19.0.4",
+    "@octokit/rest": "^21.1.1",
```

Comment:
I think other libraries of ours are using v20 because newer deps that use v21 are requiring ESM. We should probably pick v20 and related major versions